FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/response/model/node/MsgBlock.kt

```diff
@@ -86,13 +93,28 @@ interface MsgBlock : Parcelable {
         override fun newArray(size: Int): Array<Type?> = arrayOfNulls(size)
       }
 
+      @JvmField
```

Comment:
I think using `@JvmField` is redundant over the whole codebase(except a case when we really need that, for example, `Parcelable` implementation).